C_image_processing/filters/sepia_filter.c

```diff
@@ -17,58 +19,63 @@ int sepia_filter(const char *inputFile, const char *outputFile)
         return 1;
     }
 
-    // Read header info of image
-    unsigned char headerInfo[54];
-    for (int i = 0; i < 54; i++)
+    // read header info of image
+    for (i = 0; i < 54; i++)
     {
-        headerInfo[i] = getc(fileIn);
+        byte[i] = getc(fileIn);
     }
 
-    // Write header info to output file
-    fwrite(headerInfo, sizeof(unsigned char), 54, fileOut);
```

Comment:
The reason I left it as  " headerInfo " was to get rid of the comment as code would state what it does or acts as, though as the comment is reasonably small it is fine, but leaving it as a reminder, not necessary to change just a discussion and suggestion.

Reply:
Yeah, it's the better way of doing it. You remove the comment and it makes it more readable. 100% should do this in any files extracting header information from the bitmap.